packages/serverpod/lib/src/server/serverpod.dart

```diff
@@ -401,14 +411,17 @@ class Serverpod {
     _passwordManager = PasswordManager(runMode: runMode);
     _passwords = _passwordManager.loadPasswords();
 
-    // Load config
-    this.config = config?.copyWith(runMode: runMode) ??
-        ServerpodConfig.load(
-          runMode,
-          _commandLineArgs.getRaw<String>(CliArgsConstants.serverId),
-          _passwords,
-          commandLineArgs: _commandLineArgs.toMap(),
-        );
+    try {
+      this.config = config?.copyWith(runMode: runMode) ??
+          ServerpodConfig.load(
+            runMode,
+            _commandLineArgs.getRaw<String>(CliArgsConstants.serverId),
+            _passwords,
+            commandLineArgs: _commandLineArgs.toMap(),
+          );
+    } on PasswordMissingException catch (e) {
```

Comment:
What do you think of having "missing critical" exceptions (in Dart parlance they should probably be errors, as they can not be fixed without a "code" change (though in our case it's the config…)) just extend `ExitException` ?

That way we don't have to special case and remap in the cases where such an exception might be thrown.

Reply:
Yeah I think this is a good idea :) Thanks!